src/libs/actions/Policy/Tag.ts

```diff
@@ -449,9 +449,16 @@ function deletePolicyTags(policyID: string, tagsToDelete: string[]) {
     API.write(WRITE_COMMANDS.DELETE_POLICY_TAGS, parameters, onyxData);
 }
 
-function clearPolicyTagErrors(policyID: string, tagName: string, tagListIndex: number) {
-    const tagListName = PolicyUtils.getTagListName(allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`], tagListIndex);
-    const tag = allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`]?.[tagListName].tags?.[tagName];
+type ClearPolicyTagErrorsProps = {
+    policyID: string;
+    tagName: string;
+    tagListIndex: number;
+    policyTags: OnyxEntry<PolicyTagLists>;
+};
+
+function clearPolicyTagErrors({policyID, tagName, tagListIndex, policyTags}: ClearPolicyTagErrorsProps) {
```

Comment:
I'm worried that someone might move the type definition higher (while refactoring the file in the future or something) and reading the types will become harder

```suggestion
function clearPolicyTagErrors({
    policyID,
    tagName,
    tagListIndex,
    policyTags,
}: {
    policyID: string;
    tagName: string;
    tagListIndex: number;
    policyTags: OnyxEntry<PolicyTagLists>;
})
```

How about chaning the definition to the inline type? (I'm not too convinced that this approach is better but just wondering)

Reply:
@war-in thanks for the review.

I was wondering about that too, but Błażej told me to do it this way because we have a rule for that in contributingGuides.

https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#no-inline-prop-types

Reply:
Thanks for linking the guide! I see it applies to the components only (at least in the guide) but I don't have strong feelings about it, so it's up to you :)